src/Digdir.Domain.Dialogporten.Application/Features/V1/EndUser/Dialogs/Queries/Search/SearchDialogQueryValidator.cs

```diff
@@ -22,6 +23,12 @@ public SearchDialogQueryValidator()
             .Must(x => !x.ServiceResource.IsNullOrEmpty() || !x.Party.IsNullOrEmpty())
             .WithMessage($"Either {nameof(SearchDialogQuery.ServiceResource)} or {nameof(SearchDialogQuery.Party)} must be specified.");
 
+        RuleForEach(x => x.Party)
+            .Must(x => x is null || EndUserIdentifier.IsValid(x) || OrganizationIdentifier.IsValid(x))
+            .WithMessage(
+                "'{PropertyName}' must be on format 'urn:altinn:organization:identifier-no::{{norwegian org-nr}}' or " +
+                "'urn:altinn:person:identifier-no::{{norwegian f-nr/d-nr}}' with valid numbers respectively.");
```

Comment:
Kan denne trekkes ut en konstant og gjenbrukes?
Ser det er samme i de 3 validatorene, kunne de og brukt `urn:altinn:organization:identifier-no` etc. fra `DecisionRequestHelper.cs`? Eller blir det mye knot? 😅

Reply:
Godt poeng! DecisionRequestHelper ligger i Infrastructure, så det går vel litt på akkord med arkitekturen å hente det derfra, men jeg kan jo evt. lage konstanter for dette i henholdsvis EnduserIdentifier og OrganizationIdentifier, så kan vi kutte noe ned på antallet steder denne idenfieren er definert